src/ocl/convolutionocl.cpp

```diff
@@ -79,9 +79,9 @@ MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_COMPILE_ONLY)
 #if MIOPEN_USE_GEMM
 #ifdef CPPCHECK
 // Keep the value unknown in cppcheck since this can differ between opencl and hip
-static bool IsUseRocBlas;
+static bool IsBF16PathValid;
```

Comment:
@atamazov 
Please notice here that bf16 is considered valid as long as gemm is used. Under this condition we should test bf16 gemm path, including conv3d.